action/protocol/vote/protocol.go

```diff
@@ -39,6 +40,25 @@ type Protocol struct {
 // NewProtocol instantiates the protocol of vote
 func NewProtocol(cm protocol.ChainManager) *Protocol { return &Protocol{cm: cm} }
 
+// Initialize initializes the rewarding protocol by setting the original admin, block and epoch reward
+func (p *Protocol) Initialize(ctx context.Context, sm protocol.StateManager, addrs []string) error {
```

Comment:
We could restrict all new candidate logic in the poll protocol: an protocol internal switch to either honor beacon chain or load pseudo candidates.